packages/runtime/src/useCases/common/validation/ValidatableStrings.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * @pattern did:e:[a-zA-Z0-9.-]+:dids:[0-9a-f]{22}
+ * @pattern did:e:((([a-z0-9]+(-[a-z0-9]+)*)\.)+[a-z]{2,}|localhost):dids:[0-9a-f]{22}
```

Comment:
wtb uppercase letters? you should re-add them.

Reply:
whoops